list.rb

```diff
@@ -0,0 +1,25 @@
+$LOAD_PATH << '.'
+require 'my_enumerable'
```

Comment:
- [ ] I can see you try to require the `MyEnumerable` class file here, however, there is no `my_enumerable` in your repository, what you have is `enumerable`. Kindly fix it.